src/promptflow-core/promptflow/_core/run_tracker.py

```diff
@@ -392,13 +392,12 @@ def set_inputs(self, run_id: str, inputs: Mapping[str, Any]):
         run_info = self.ensure_run_info(run_id)
         run_info.inputs = inputs
 
-    def set_openai_metrics(self, run_id: str):
+    def set_openai_metrics(self, run_id: str, api_calls: List):
```

Comment:
We can simply do not use run_id for this scenario?

Reply:
Refined the code to avoid changing the interface.